pkg/ioutils/writers_test.go

```diff
@@ -6,6 +6,32 @@ import (
 	"testing"
 )
 
+func TestWriteCloserWrapperClose(t *testing.T) {
+	called := false
+	writer := bytes.NewBuffer([]byte{})
+	wrapper := NewWriteCloserWrapper(writer, func() error {
+		called = true
+		return nil
+	})
+	err := wrapper.Close()
+	if err != nil {
```

Comment:
or `c.Assert(wrapper.Close(), check.IsNil)` (just learnt thanks to @tiborvass :smile: )